tests/ext/sandbox/dd_trace_function_alias.phpt

```diff
@@ -12,6 +12,7 @@ function bar($message)
 }
 
 dd_trace_function('bar', function (SpanData $span) {
+    $span->name = $span->resource = 'bar';
```

Comment:
A lot of the tests have been updated to explicitly set `name` and `resource`. Is there a reason for changing this from the default fallback?

Reply:
The point was that these tests were not explicitly intending to test that specific functionality. I believe tests should generally be restricted to what they are actually testing. I can revert these if you think it's best.

Reply:
No need to revert. 👍